servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcBindableService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.grpc.api;
+
+/**
+ * A <a href="https://www.grpc.io">gRPC</a> service that can generate factory instances bound to itself.
+ *
+ * @param <F> Filter type
+ * @param <S> Service Type
+ * @param <FF> FilterFactory type
```

Comment:
The issue is that then they look like class names rather than generic type names which is especially misleading since the names are (almost) valid class names. I can make it consistent with GrpcServiceFactory but it does seem odd.

Reply:
Final thoughts on this?

Reply:
Both ways work fine for me, I was looking for consistency inside grpc module. But if we consider the whole project, we are not really consistent and sometimes use letters, sometimes full names for generics.
The full name is nice because you don't need to remember what is what (IDE highlights generic types differently, so people don't consider them as class names). But for the tiny interface like this it doesn't matter.